yarn.lock

```diff
@@ -8228,7 +8228,7 @@ anymatch@^3.0.0, anymatch@~3.1.2:
     normalize-path "^3.0.0"
     picomatch "^2.0.4"
 
-anymatch@^3.0.3, anymatch@~3.1.1:
+anymatch@^3.0.3:
```

Comment:
I don't think I'm seeing any changes to `package.json`, might wanna revert this file